rackcheck.scrbl

```diff
@@ -486,6 +487,18 @@ Don't use them to produce values for your tests.
   ]
 }
 
+@defproc[(property-name [property any/c]) any/c?]{
+  Returns name of the @racket[property].
+
+  @ex[
+    (define-property prop-list-map-identity
+     ([xs (gen:list gen:natural)])
+     (check-equal? (map identity xs) xs))
```

Comment:
Nitpick: `racket/base` contains `values`, which is an identity function itself. I tend to consider `racket/function` a bit unidiomatic. Would you mind dropping the module here and in the example module above and using `values` instead?

Reply:
Done: https://github.com/Bogdanp/rackcheck/pull/9/commits/62e29c6603eb49048bee5636a414c82c41e80c2a and here https://github.com/Bogdanp/rackcheck/pull/9/commits/6f5501302f69479563f8a430ebc88ddbec8aa026